db/migrate/20201130093110_devise_create_users.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users, id: :uuid do |t|
+      t.string :first_name,        null: false
+      t.string :last_name,         null: false
+
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
Do we want this to have a default?

Reply:
That's what devise generated. I could remove the default but in practice it wouldn't make any difference